### PR TITLE
7163 - Change header pseudo element to actual svg icon

### LINKED ISSUE #7163: Header: Change from Pseudo Elements to actual icon / Header Cleanup
**Describe the bug**
Change header hamburger to use the icon and change header text size.

**To Reproduce**
1. Go to https://main-enterprise.demo.design.infor.com/components/header/example-index.html
2. Notice the icon is not a real icon in the design system
3. See text size

**Expected behavior**
- Change all hamburger examples to use the actual SVG icon
- Change header size to 24px
- What about double text (two sizes) https://main-enterprise.demo.design.infor.com/components/header/example-eyebrow.html

**Additional context**
The icon was using https://main-enterprise.demo.design.infor.com/components/icons/example-pseudo-elements.html / keep the page for now. But no longer need to use it on header

https://jira.infor.com/browse/IDS-887 

**Conform To Design**
https://www.figma.com/file/yaJ8mJrqRRej8oTsd6iT8P/IDS-(SoHo)-Component-Library-v4.5?node-id=26118%3A55002&t=0KTc3uMUCKmOZ3a0-1

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR changes the header from a pseudo-element to an actual svg icon. Adjusted the width, height, and position of the notification dots.

**Related github/jira issue (required)**:

Closes #7163

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

1.) Application Menu
- Go to
  - http://localhost:4000/components/applicationmenu/example-index
  - http://localhost:4000/components/applicationmenu/example-performance
  - Check the header/hamburger-menu
  - It should use the actual svg icon
  - The SVG icon should be `24x24`

2.) Builder
- Go to
  - http://localhost:4000/components/builder/example-builder-nosidebar (Resize the page to show the arrow back button)
  - http://localhost:4000/components/builder/example-builder-subtitle-test (No need to resize)
  - http://localhost:4000/components/builder/example-builder (Resize the page to show menu button)
  - http://localhost:4000/components/builder/example-builder-wizard (Resize the page to show menu button)
  - Inspect the element
  - It should use SVG icon and the size should be `24x24`

3.) Button
- Go to http://localhost:4000/components/button/example-badge
  - Inspect the appmenu icon with notification dots
  - It should use SVG icon and size should be `24x24`
  - Notification dots should be perfectly aligned in different positions
  - Test in classic 

4.) Form Compact
- Go to http://localhost:4000/components/form-compact/example-list-detail.html
- Inspect the menu button
- It should use SVG icon and size should be `24x24`

5.) Header
- Go to
  - http://localhost:4000/components/header/example-emphasize-id
  - http://localhost:4000/components/header/example-emphasize-page-title
  - http://localhost:4000/components/header/example-eyebrow
  - http://localhost:4000/components/header/example-record-id
  - http://localhost:4000/components/header/test-header-gauntlet
  - Inspect the back button/app menu button
  - It should use SVG icon and size should be `24x24`

6.) Page Patterns
- Go to
  - http://localhost:4000/components/page-patterns/example-list-detail-search-paging
  - http://localhost:4000/components/page-patterns/example-list-detail
  - http://localhost:4000/components/page-patterns/test-landmark-requisitions-screen
  - Inspect the back button/app menu button
  - It should use SVG icon and size should be `24x24`

7.) Toolbar-flex
- Go to
  - http://localhost:4000/components/toolbar-flex/test-gauntlet
  - Check the appmenu trigger element
  - It should use SVG icon and size should be `24x24`
  - http://localhost:4000/components/toolbar-flex/test-toolbar-nav-badge
  - Inspect the appmenu icon with notification dots
  - It should use SVG icon and size should be `24x24`
  - Notification dots should be perfectly aligned in different positions
  - Test in classic 

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
- [x] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@
 - `[Fileupload]` Icon adjustments in compact mode. ([#7149](https://github.com/infor-design/enterprise/issues/7149))
 - `[Fileupload]` Icon adjustments in classic mode. ([#7265](https://github.com/infor-design/enterprise/issues/7265))
 - `[Header]` Fixed a bug in subheader where the color its not appropriate on default theme. ([#7173](https://github.com/infor-design/enterprise/issues/7173))
+- `[Header]` Changed the header from pseudo elements to actual icon. ([#7163](https://github.com/infor-design/enterprise/issues/7163))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:38]: Can you add a brief not
```suggestion
- `[Header]` Changed the header from pseudo elements to actual icon. Please make the follow [change to your app menu icon](https://github.com/infor-design/enterprise/pull/7285/files#diff-4ee8ef8a5fe8ef128f558004ce5a73d8b2939256ea3c614ac26492078171529bL3-R5) to get the best output.  ([#7163](https://github.com/infor-design/enterprise/issues/7163))
```

**Review by tmcconechy — DISMISSED** [review]
Looks good. Just wanted to try to add a comment that tells people they need to make a change.

**Comment by yohannahbautista** [comment]
Hello, will the css for the classes one, two, three still be needed or can it be removed?

**Comment by ericangeles** [comment]
@yohannahbautista for now we just leave it there before everyone can adopt the new change.

**Review by yohannahbautista — APPROVED** [review]

**Pull request merged**